webpack/webpack.common.js

```diff
@@ -38,6 +38,10 @@ module.exports = {
                 loader: 'babel-loader',
                 exclude: /node_modules/,
             },
+            {
```

Comment:
This rule allows for requiring images. See https://webpack.js.org/guides/asset-modules/ for details.